Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -540,3 +540,12 @@ def gfonts_repo_structure(fonts):
   #        about the expected structure.
   abspath = get_absolute_path(fonts[0])
   return abspath.split(os.path.sep)[-3] in ["ufl", "ofl", "apache"]
+
+
+@condition
+def production_metadata():
+  """Get the Google Fonts production metadata"""
+  import json
+  import requests
+  meta_url = "http://fonts.google.com/metadata/fonts"
+  return json.loads(requests.get(meta_url).text[5:])
```

Comment:
Why `[5:]`?
It is not clear to me what's going on here, so I think there should be a comment here to clarify your purpose.